atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ProfilingKeyValueService.java

```diff
@@ -45,6 +48,8 @@
 import com.palantir.util.paging.TokenBackedBasicResultsPage;
 
 public class ProfilingKeyValueService implements KeyValueService {
+    public static final String SLOW_LOGGER_NAME = "Profiling KVS Slow-Operation Logger";
```

Comment:
Sorry I now see the variable is used in the tests, let's keep it package private then.